src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsrestrictedpatientsapi/service/SubjectAccessRequestServiceTest.kt

```diff
@@ -34,13 +34,13 @@ class SubjectAccessRequestServiceTest {
         Optional.of(makeRestrictedPatient()),
       )
 
-      val restrictedPatient = service.getContentFor("A12345", null, null)?.content
+      val restrictedPatient = service.getPrisonContentFor("A12345", null, null)?.content
 
-      assertThat(restrictedPatient?.prisonerNumber).isEqualTo("A12345")
-      assertThat(restrictedPatient?.supportingPrisonId).isEqualTo(PRISON.agencyId)
-      assertThat(restrictedPatient?.hospitalLocationCode).isEqualTo(HOSPITAL.agencyId)
-      assertThat(restrictedPatient?.commentText).isEqualTo("test")
-      assertThat(restrictedPatient?.dischargeTime).isEqualTo(LocalDateTime.parse("2020-10-10T20:00:01"))
+      assertThat(restrictedPatient).extracting("prisonerNumber").isEqualTo("A12345")
```

Comment:
seemed a bit nicer to do this rather than casting the contents to my implementation